src/components/Banner/Banner.tsx

```diff
@@ -44,12 +44,20 @@ export interface Props {
 
 export default class Banner extends React.PureComponent<Props, never> {
   static contextTypes = contentContextTypes;
+  static childContextTypes = contentContextTypes;
+
   private wrapper = React.createRef<HTMLDivElement>();
 
   public focus() {
     this.wrapper.current && this.wrapper.current.focus();
   }
 
+  getChildContext() {
+    return {
+      withinContentContainer: true,
```

Comment:
Should we introduce a new context that specific to components where links should be rendered with monochromatic styles?

Reply:
This question came up after looking at these results from Percy: https://percy.io/Shopify/polaris-react/builds/2009434

Reply:
I would say so, `withinContentContainer` is used for Banner's at the moment

Reply:
Thanks for the context 👍 I added a new `withinBanner` context to use here instead.